recipe/meta.yaml

```diff
@@ -1,9 +1,9 @@
-{% set version = "2.2.6" %}
-{% set dev = "" %}
+{% set version = "2.3.0" %}
+{% set dev = "rc1" %}
```

Comment:
```suggestion
{% set version = "2.3.0" %}
{% set dev = "" %}
```

Reply:
I'd click the commit button, except I am unsure if the change to add the `rc` channel (commit 335317fa318ab2d5be65d2c2fb894590bcffa8a7) needs to be reverted